test/auth/config.ts

```diff
@@ -38,6 +38,12 @@ export default buildConfigWithDefaults({
           saveToJWT: true,
           hasMany: true,
         },
+        {
+          name: 'namedSaveToJWT',
```

Comment:
I find this example semantically hard to follow, do you think we could adjust the names that we use here? i.e. `fieldName` & `jwtKey` or similar?